core/server/server.js

```diff
@@ -90,10 +90,10 @@ const publicConfigSchema = Joi.object({
         .integer()
         .min(1)
         .when('enabled', { is: true, then: Joi.required() }),
-      intervalSeconds: Joi.number()
```

Comment:
Actually, nothing changes here, but after few commits Prettier formatted it this way.